<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenInfo</class>
 <widget class="QMainWindow" name="FenInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="systemTab">
       <attribute name="title">
        <string>System</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QScrollArea" name="scrollArea_6">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>855</width>
             <height>395</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="osVersion">
              <property name="text">
               <string>os version</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="osDirectory">
              <property name="text">
               <string>Répertoire windows</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="osNbProc">
              <property name="text">
               <string>nombre de processus</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="cpuTab">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QScrollArea" name="scrollArea_5">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>855</width>
             <height>395</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="cpuName">
              <property name="text">
               <string>Nom du cpu</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="cpuNbCore">
              <property name="text">
               <string>Nombre de processor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="cpuSpeed">
              <property name="text">
               <string>Vitesse actuelle du processeur</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="cpuVoltage">
              <property name="text">
               <string>Voltage Actuel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="cpuUsed">
              <property name="text">
               <string>Utilisation du Cpu</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="memoireTab">
       <attribute name="title">
        <string>Mémoire</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTableWidget" name="tableMemoire">
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>120</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Lettre de lecteur</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type de lecteur</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Espace Disponible</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Espace Total</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Vitesse en lecture</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Vitesse en écriture</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>System de fichier</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="cgTab">
       <attribute name="title">
        <string>Carte graphique</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>855</width>
             <height>395</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLabel" name="nomCg">
              <property name="text">
               <string>nom des cartes graphiques présentes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="graphicInfo">
              <property name="text">
               <string>info sur l'affichage</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="graphicBits">
              <property name="text">
               <string>nombre de bits par pixels</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="graphicRefresh">
              <property name="text">
               <string>Rafraichissement de l'écran</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="csTab">
       <attribute name="title">
        <string>Carte son</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QTableWidget" name="tableSound">
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>400</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>250</number>
          </attribute>
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Nom du matériel</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Identifiant</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="driverTab">
       <attribute name="title">
        <string>Driver</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(245, 245, 245);


</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>855</width>
             <height>395</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="driverNom">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Nom des drivers présents</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
